wiki/Client/Beatmap_editor/Distance_snap/de.md

```diff
@@ -1,11 +1,21 @@
 ---
-outdated_translation: true
+tags:
+  - DS
+  - time-distance equality
 ---
 
 # Distance Snap
 
-![Beispiel](img/Beatspacing.jpg "Beispiel")
+::: Infobox
+![](/wiki/shared/editor/editor-draw-distsnap.png "Das Distance Snap Icon")
+:::
 
-Der einfachste Weg, um sicherzustellen, dass der Abstand der Beats korrekt ist, ist die Distance Snap Funktion im Editor zu benutzen, welches Sie auf der rechten Seite finden können. Der jeweilige Abstand zwischen zweier Hit Objekten lässt sich durch das Gedrückt Halten der ALT-Taste und das Drehen des Mausrades ändern. Dadurch soll gewährleistet werden, dass Beats, die in der Timeline nah aneinander sind, auch auf der Beatmap nah aneinander und andersherum. Dies hilft beim intuitiven Spielen, sowie guten Beat-Abständen. Korrekte Beat-Abstände sind garantiert, wenn Sie im ganzen Lied den Distance Snap durchgehend verwenden!
+**Distance Snap** ist eine Funktion des Beatmap Editors, welche den Abstand von Objekten auf dem Spielfeld limitiert. Der Abstand zwischen zwei Objekten basiert bei Verwendung auf dem Zeitraum zwischen beiden Objekten und wird aus der [Slider Velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) und dem Distance Spacing Multiplikator berechnet.
```

Comment:
Are those the official terms? I've been a german mapper for 12 years and all I've seen used and used are the english terms...

I'm generally fine with those changes I just think it's strange to translate terms that end up being ignored by the people using the editor mostly. Because that could make the article less useful if you want to look that stuff up

Reply:
I know that it's not an established term, however, I like to experiment with translations and see what is accepted and what not to have as much German in the article as possible.

By the way, I opened a poll to see what others think of this on the German discord.

How about mentioning the English variant in brackets?